docs/recipes/IsolatingSubapps.md

```diff
@@ -0,0 +1,74 @@
+Consider the case of a "big" app that embeds smaller "subapps":
```

Comment:
Let’s add a title and keep the doc format consistent with the rest of the recipes.